packages/pg-v5/test/lib/psql.js

```diff
@@ -29,37 +37,48 @@ const bastionDb = {
 }
 
 describe('psql', () => {
+  let fakePsqlProcess, fakeTunnel, tunnelStub
+  let sandbox
+  let mockSpawn
+  let psql, bastion
+
   beforeEach(() => {
-    sinon.stub(Math, 'random').callsFake(() => 0)
```

Comment:
this was changed to use the `sinon.sandbox` so that we don't have to manually clean it up in the `afterEach`